packages/@aws-cdk/aws-batch/README.md

```diff
@@ -119,7 +119,7 @@ The alternative would be to use the `BEST_FIT_PROGRESSIVE` strategy in order for
 
 Simply define your Launch Template:
 
-```text
+```ts
```

Comment:
I don't remember exactly why this example was marked as "typescript only", but if it passes `jsii-rosetta` then I'm fine with changing this now. However, we need to remove the comment that states `// This example is only available in TypeScript`